android/src/main/java/com/rnappauth/RNAppAuthModule.java

```diff
@@ -762,7 +763,11 @@ private void warmChromeCustomTab(Context context, final String issuer) {
             @Override
             public void onCustomTabsServiceConnected(ComponentName name, CustomTabsClient client) {
                 client.warmup(0);
```

Comment:
Did you know that .warmup can throw? We experienced crashes here before: https://github.com/ankidroid/Anki-Android/commit/6503e5dda718397a41de9c90e14a0a7ded0e0e37

https://github.com/ankidroid/Anki-Android/pull/6146/files#diff-b4eaf9bae4ba5cd94138b46ef441e244R154

You have to catch IllegalStateException + SecurityException, or just Exception really

The crashes are rare but they do come in